lib/ads/interstitial_ad.js

```diff
@@ -94,6 +98,11 @@ shaka.ads.InterstitialAd = class {
    * @export
    */
   isSkippable() {
+    if (this.isSkippable_ && this.skipFor_ != null) {
+      const position = this.getDuration() - this.getRemainingTime();
+      const maxTime = this.skipOffset_ + this.skipFor_;
+      return position < maxTime;
```

Comment:
So, if you don't skip fast enough, you lose the chance?  That seems backwards to how most skippable ads work, in my experience.

Reply:
Yes, it is a new feature that is only activated for HLS interstitials, and it is configurable. If you configure X-ENABLE-SKIP-FOR with a number high or equal to the duration of the ad or omit this parameter, you have the old behavior.